packages/editor/src/components/properties/EnvMapBakeNodeEditor.tsx

```diff
@@ -145,20 +145,20 @@ export const EnvMapBakeNodeEditor: EditorComponentType = (props) => {
   }
 
   const onChangePosition = (value) => {
-    bakeComponent.bakePositionOffset.copy(value)
+    bakeComponent.bakePositionOffset.set(value)
```

Comment:
these do not work. Vector3 and other math objects need to have `value.copy` here instead of `set`

Reply:
i didnt know that , reverting

Reply:
i just read up, the bakeComponent.bakePositionOffset is of type State<Vector3,{}> and not Vector3 like when we were using Getcomponent
i think i should be able to use set

Reply:
ill test it

Reply:
the set one isnt working, the older one worked